add cluster prepare to c16x docs (#1406)
replicatedhq/replicated-docs: docs/vendor/testing-how-to.md

## Creating and Preparing Clusters

The Replicated compatibility matrix functionality is provided through the replicated CLI  `cluster` commands.

You can run the commands to manually create a cluster when you need one for a short period of time, such as when debugging a support issue or to use testing as part of an inner development loop. You can also integrate the commands in your continuous integration and continuous delivery (CI/CD) workflows to automatically provision clusters for running tests. For more information, see [Integrating with CI/CD](ci-overview). 

You can use both `cluster create` and `cluster prepare` to provision clusters. The following describes the use cases for each command:

* `cluster create`: Provisions a cluster based on the parameters specified. After a cluster is provisioned, an application can be installed in the cluster by creating a release, a temporary channel, and a temporary customer in the Replicated platform. A recommended use case for the `cluster create` command is provisioning clusters for testing in continuous delivery (CD) workflows that release your software to customers.

>   The following example creates a kind cluster with Kubernetes version 1.25.2, a disk size of 100 GiB, and an instance type of `r1.small`. 

Reviewer: ```suggestion
  The following example creates a kind cluster with Kubernetes version 1.27.0, a disk size of 100 GiB, and an instance type of `r1.small`. 
```

  ```bash

>   replicated cluster create --name kind-example --distribution kind --version 1.25.2 --disk 100 --instance-type r1.small

Reviewer: ```suggestion
  replicated cluster create --name kind-example --distribution kind --version 1.27.0 --disk 100 --instance-type r1.small
```

  ```

  For command usage, see [cluster create](/reference/replicated-cli-cluster-create) in the _replicated CLI_ reference.

> * `cluster prepare`: Provisions a cluster based on the parameters specified and then installs an application in the cluster. The `cluster prepare` command allows you to install an application in a cluster for testing without needing to create a release, a temporary channel, or a temporary customer in the Replicated platform. A recommended use case for the `cluster prepare` command is provisioning clusters for testing in continuous integration (CI) workflows that run on every commit.

Reviewer: ```suggestion
* `cluster prepare`: Provisions a cluster based on the parameters specified and then installs an application in the cluster. The `cluster prepare` command allows you to install an application in a cluster for testing without needing to create a temporary channel, or a temporary customer in the Replicated platform. A recommended use case for the `cluster prepare` command is provisioning clusters for testing in continuous integration (CI) workflows that run on every commit.
```

It does create a release. However, the release creation is part of `cluster prepare`. Which is different from `cluster create`, where there is no release creation.
